Volume/Views/MainView/HomeList.swift

```diff
@@ -146,8 +132,9 @@ struct HomeList: View {
 
                 // Invisible navigation link only opens if application is opened
                 // through deeplink with valid article
-                if let url = onOpenArticleUrl {
-                    NavigationLink("", destination: BrowserView(article: url, navigationSource: .morePublications), isActive: $openedUrl)
+                if let articleID = onOpenArticleUrl {
+                    let _ = print("openArticleUrl change detected")
```

Comment:
Why didn't you just print here?

Reply:
From [this reply](https://github.com/cuappdev/volume-ios/pull/67#discussion_r753451649): 
> SwiftUI considers a print statement to return a () value, which isn't allowed in the ViewBuilder body. Also, I forgot to remove this print 😅